src/connection.ts

```diff
@@ -96,9 +98,9 @@ export class Connection {
 
     this._hasSessionId = connections !== null;
     if (this._hasSessionId) {
-      this._sessionIds = Array(connections)
-        .fill('')
-        .map(() => factoryId());
+      this._sessionIds = Array.from({length: connections || 0}).map(() =>
+        factoryId(),
```

Comment:
This was just to make TypeScript happy... i think it's not really relevant from a JS perspective...